setup.py

```diff
@@ -168,7 +168,6 @@ def build_extensions(self):
 requirements = [
     "numpy",
     "pennylane>=0.15",
-    "pybind11",
```

Comment:
Note this removal. This means that all the dependencies of `lightning` are also dependencies of PennyLane.

Reply:
Just so I'm clear, this change requires the user to ensure they have pybind11 available manually, rather than as an auto-pulled dep when we try to build? As in, explictly running `pip install -r requirements.txt` is needed?

Reply:
Yes, agreed!

Reply:
There might be one additional use-case, not sure how common it is though:

> A user wants to `pip` install lightning on an unsupported system and compile from source.

In this case, the user would have to:

1. Make sure all C++ dependencies are installed
2. Run

   ```
   pip install pybind11
   pip install pennylane-lightning --no-binary :all:
   ```

Potentially this could occur in Docker workflows. Say a user is using the Alpine Linux image that uses `musl`; (as far as I am aware?) no wheels are available for this platform, but they

- would like to avoid using git to reduce the docker image size
- would like to use the C++ simulator

Reply:
Good idea, have updated the instructions: 8e6ed64364470875634c77c60d23e0fc3190a41f